docs/UserGuide.md

```diff
@@ -132,39 +141,59 @@ each sub category when sorted.
 
 <div style="page-break-after: always;"></div>
 
-## 4. Features <a name="features"></a>
+## 4. Short Tutorial <a name="tutorial"></a>
+
+Excited to try **MODTrek** out? Here is a sequence of basic commands to help you start tracking your modules:
+
+1. `add /m CS1101S /c 4 /y Y1S1`: Adds the module CS1101S to the list. The Results Section should display the Module List, with a Module Card created for CS1101S, placed within a Module Group categorised as Y1S1 (sorted by year-sem by default).
```

Comment:
I have discussed with @Cheamybunny and we have decided that there's no need to introduce the ModuleList, ModuleGroup and ModuleCard components in the UG as they are intuitive enough to the user using the GUI. It is originally only meant for the DG to connect each UI component to its corresponding Java class.

In UG, we will just introduce the three screens of the Results Section: Degree Progress Screen, Module List Screen,  Module Search Screen. @Cheamybunny will reflect this change in a future commit.

So this part can be updated to reflect this change.